ui/src/components/App.scss

```diff
@@ -1,17 +1,63 @@
-@import '../index.scss';
+@import './common/CardTable.scss';
+@import './common/DonutChart.scss';
+@import './common/FormErrorHandled.scss';
+@import './common/Holdings.scss';
+@import './common/InviteAcceptTile.scss';
+@import './common/LandingPage.scss';
+@import './common/Notification.scss';
+@import './common/OnboardingTile.scss';
+@import './common/OrderCard.scss';
+@import './common/OverflowMenu.scss';
+@import './common/Page.scss';
+@import './common/PageSection.scss';
+@import './common/Profile.scss';
+@import './common/StripedTable.scss';
+@import './common/TabViewer.scss';
+@import './common/TopMenu.scss';
+@import './common/Wallet.scss';
+@import './common/WalletTransaction.scss';
+@import './common/WelcomeHeader.scss';
+
 @import './LoginScreen.scss';
+@import './RoleSelectScreen.scss';
+
+@import './Broker/Broker.scss';
 @import './Investor/Investor.scss';
 @import './Issuer/Issuer.scss';
-@import './common/TopMenu.scss';
-@import './Broker/Broker.scss';
 
-.app {
-    height: 100vh;
-    width: 100vw;
+@mixin app {
+    .app {
+        height: 100vh;
+        width: 100vw;
+
+        /* common css */
+        @include card-table;
+        @include donut-chart;
+        @include form-error-handled;
```

Comment:
After some consideration, I had settled on importing and `@include`ing all the common styles centrally at the top app level, since they get applied downwards

Reply:
i like this approach